.github/workflows/continous-integration.yml

```diff
@@ -285,6 +286,136 @@ jobs:
         COVERALLS_SERVICE_NAME: github
       run: poetry run coveralls
 
+  integration_test:
```

Comment:
potentially out of the scope of this PR, but: shouldn't we run the integration tests against different Python versions / Windows as well?

Reply:
Definitely out of scope 👍🏻 but to add to a TODO / future discussions